src/test/java/org/apache/commons/io/FileUtilsDirectoryContainsTestCase.java

```diff
@@ -134,9 +134,16 @@ public void testSameFile() throws IOException {
     }
 
     @Test
-    public void testIO466() throws IOException {
```

Comment:
Please don't remove an existing test, it makes reviewing harder.

Reply:
The fix has been applied in d42b9286eb7a534f1b2b6243858ff7ef6fcac14c so isn't it time to close this pr?
anyway, I've rebased and restored the test.